core/src/main/resources/error/delta-error-classes.json

```diff
@@ -1382,16 +1382,7 @@
     "message" : [
       "",
       "Your current table protocol version does not support changing column mapping modes",
-      "using <config>.",
-      "",
-      "Required Delta protocol version for column mapping:",
-      "<requiredVersion>",
-      "Your table's current Delta protocol version:",
-      "<currentVersion>",
```

Comment:
Can we still keep this? I think it is useful to include <currentVersion> in the error message.

We can just add <advice> in place of the last line "Please upgrade your ..." possibly?

Reply:
Something like this .?

```
Your current table protocol version does not support changing column mapping modes
using delta.columnMapping.mode.

Required Delta protocol version for column mapping:
Protocol(2,5)
Your table's current Delta protocol version:
Protocol(2,6)

Please upgrade your Delta table to reader version 2 and writer version 5
and change the column mapping mode to 'name' mapping. You can use the following command:

ALTER TABLE <table_name> SET TBLPROPERTIES (
   'delta.columnMapping.mode' = 'name',
   'delta.minReaderVersion' = '2',
   'delta.minWriterVersion' = '5')

```

Reply:
yep that would be great!

Reply:
resolving the change.!